src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private final Random random = new Random();
+
+    private final int limitCountFigure = 5;
+    private final int limitMaxSide = 5;
```

Comment:
these are not constants - constants should be static and have names in UPPER_CASE